### PR TITLE
Minor edits to the documentation

### PR DESCRIPTION
**Pull request by sosna**
Hey there,

First, thanks for your nice project :).

While reading the documentation, I found a few typos or missing punctuation, and thought you might want to receive a pull request for these.

Cheers,
Xavier

PS: I read the contributing.md, but it does not seem to apply to changes to the documentation?

### CODE REVIEW

File: docs/running-without-web-ui.rst [hunk 1]
```diff
@@ -83,4 +83,4 @@ Below is an example that'll set the exit code to non zero if any of the followin
         else:
             environment.process_exit_code = 0
 
-(this code could go into the locustfile.py or in any other file that is imported in the locustfile)
+This code could go into the locustfile.py or in any other file that is imported in the locustfile.
```
Comment (cyberw) [inline docs/running-without-web-ui.rst:86]: The parenthesis do actually make a difference. Maybe change it to ”Note that …”

File: docs/testing-requests-based SDK's.rst [hunk 2]
```diff
@@ -4,10 +4,10 @@
 Testing Requests based SDKs
 =============================
 
-If a prebuilt SDK is available for your target system. Locust has a supported pattern for integrating
+If a prebuilt SDK is available for your target system, Locust has a supported pattern for integrating
 its usage into your load testing efforts.
 
-The only perquisite to achieve this; is that the SDK needs to have an accessible ``request.Sessions``
+The only perquisite to achieve this is that the SDK needs to have an accessible ``request.Sessions``
```
Comment (cyberw) [inline docs/testing-requests-based SDK's.rst:10]: while you are at it, its spelled ”prerequisite” :)

File: docs/writing-a-locustfile.rst [hunk 3]
```diff
@@ -293,7 +293,7 @@ The easiest way to add a task for a User is by using the :py:meth:`task <locust.
             print("User instance (%r) executing my_task" % self)
 
 **@task** takes an optional weight argument that can be used to specify the task's execution ratio. In
-the following example *task2* will have twice the chance of being picked as *task1*:
+the following example, *task2* will be twice as likely to be selected than *task1*:
```
Comment (cyberw) [inline docs/writing-a-locustfile.rst:296]: … to be selected AS task1

File: docs/writing-a-locustfile.rst [hunk 4]
```diff
@@ -343,14 +343,14 @@ with a task that looks like this::
 
     {my_task: 3, another_task: 1}
 
-*my_task* would be 3 times more likely to be executed than *another_task*.
+*my_task* would be 3 times as likely to be executed than *another_task*.
```
Comment (cyberw) [inline docs/writing-a-locustfile.rst:346]: ”as” again

**Comment by cyberw** [comment]
I almost couldnt believe we still had that many errors. Have a look at my comments and I’ll merge.

File: docs/running-without-web-ui.rst (hunk 1 above)
Comment (sosna) [reply docs/running-without-web-ui.rst]: I made the change but let me know in case I misunderstood it :)

File: docs/running-without-web-ui.rst [hunk 5]
```diff
@@ -83,4 +83,4 @@ Below is an example that'll set the exit code to non zero if any of the followin
         else:
             environment.process_exit_code = 0
 
-This code could go into the locustfile.py or in any other file that is imported in the locustfile.
+(Note that this code could go into the locustfile.py or in any other file that is imported in the locustfile.)
```
Comment (cyberw) [inline docs/running-without-web-ui.rst:86]: You can remove the parentheses now.

**Comment by sosna** [comment]
Done :)

**Comment by cyberw** [comment]
Thanks!!

**Pull request merged**